docs/Releases.md

```diff
@@ -215,3 +215,13 @@ We do this so that our first release will have a proper patch version of 0, as s
 ```
 
 </details>
+
+## Hermes Compatibility (0.74 and above)
+
+*Note: This is only applicable if Hermes is enabled via the `USE_HERMES=1` environment variable when running `pod install`.*
```

Comment:
Speaking of.. can we find what version of RN set Hermes as the default and align to that now?

Reply:
I'm referring to setting Hermes as the default so that we don't need to do `USE_HERMES=1 pod install`. That's how RN Core has it